packages/wdio-utils/src/initialisePlugin.js

```diff
@@ -8,9 +9,9 @@ import { safeRequire } from './utils'
  */
 export default function initialisePlugin (name, type, target = 'default') {
     /**
-     * directly import packages that are scoped
+     * directly import packages that are scoped or start with an absolute path
      */
-    if (name[0] === '@') {
+    if (name[0] === '@' || name[0] === path.sep) {
```

Comment:
[path.sep](https://nodejs.org/api/path.html#path_path_sep) is either `/` or `\`.

How to handle the following cases:
- `./foo/bar` mentioned in the doc
- `c:\foo\bar`
- or just `foo\bar`